k8-operator/controllers/infisicalsecret_helper.go

```diff
@@ -225,12 +227,44 @@ func (r *InfisicalSecretReconciler) GetInfisicalServiceAccountCredentialsFromKub
 	return model.ServiceAccountDetails{AccessKey: string(accessKeyFromSecret), PrivateKey: string(privateKeyFromSecret), PublicKey: string(publicKeyFromSecret)}, nil
 }
 
+type TemplateSecret struct {
```

Comment:
Can we move this to a model file please?

Reply:
Changed